pom.xml

```diff
@@ -245,7 +245,7 @@
         <spotless.java.googlejavaformat.version>1.7</spotless.java.googlejavaformat.version>
         <spotless.python.includes></spotless.python.includes>
         <spotless.python.black.version>22.3.0</spotless.python.black.version>
-        <spotless.scala.scalafmt.version>3.7.1</spotless.scala.scalafmt.version>
+        <spotless.scala.scalafmt.version>3.7.3</spotless.scala.scalafmt.version>
```

Comment:
`.scalafmt.conf` should be updated as well, and let's add a comment to mention it here.

Reply:
done